src/migrate.js

```diff
@@ -88,6 +88,11 @@ function Migrate() {
 		}));
 	};
 
+	this.createWebhook = async function(secret, events, payloadUrl, orgName, repoName) {
```

Comment:
Do we need it async really? It should be ok, even if its return Promise for now.
Also, catch the exception and handle it here

Reply:
Done